mdnoble73/aspen-discovery
# Refactor: Summon combined results update

TEST PLAN:
1) Log in to Aspen as an administrator.
2) Ensure that the Summon module is set up and all relevant credentials have been added to the settings.
3) Navigate to the 'Primary Configuration' section in the admin menu section and select 'Library Systems.'
4) Select the option to edit your chosen library.
5) Scroll down to the 'Combined Results' section.
6) Enable combined results and click 'Add New' in the combined results section.
7) Add 'Summon' and another option and save.
8) Use the search bar to carry out a search, selecting 'Combined Results' in the dropdown.
9) Your results should return in two columns, one for your Summon results and one for your other selection.
